Dockerfile.windows

```diff
@@ -219,15 +216,11 @@ RUN `
   Write-Host INFO: Downloading compiler 3 of 3...; `
   Download-File https://raw.githubusercontent.com/jhowardmsft/docker-tdmgcc/master/binutils.zip C:\binutils.zip; `
   `
-  Write-Host INFO: Installing PS7Zip package...; `
-  Install-Package PS7Zip -Force | Out-Null; `
-  Write-Host INFO: Importing PS7Zip...; `
-  Import-Module PS7Zip -Force; `
-  New-Item C:\git -ItemType Directory | Out-Null ; `
-  cd C:\git; `
   Write-Host INFO: Extracting git...; `
-  Expand-7Zip C:\gitsetup.7z.exe | Out-Null; `
-  cd C:\; `
+  Expand-Archive C:\gitsetup.zip -DestinationPath C:\git-temp; `
```

Comment:
Using `install-package` will pull in nuget.exe (I don't know if other crud comes along with it). I could use that, but this seems leaner.

The Nuget package is released by `dscho` who's the same guy cutting releases on the GitHub repo. They're in sync and there's one release process as far as I can determine. I don't think it'll be less thruthy than the packages on GitHub.

The reason for using the NuGet package is that it can be deployed without pulling in other tools like 7zip.